docs/shared-docker.asciidoc

```diff
@@ -2,7 +2,7 @@
 ==== Run {beatname_uc} on Docker
 
 Docker images for {beatname_uc} are available from the Elastic Docker
-registry. The base image is https://hub.docker.com/_/centos/[centos:7].
+registry. The base image is https://hub.docker.com/_/ubuntu[ubuntu:20.04].
```

Comment:
I think we switched to 22.04 in Oct 2023: https://github.com/elastic/apm-server/blob/main/packaging/docker/Dockerfile#L1

Reply:
Thanks for catching this. May need 2 separate PRs then, one for main to show 22.04 and one up to 8.11 to show 20.04.